README.md

```diff
@@ -1,5 +1,10 @@
 # jshint loader for webpack
 
+## npm package
+
+``npm install --save jshint-loader jshint``
```

Comment:
Hey @nikhilknoldus, thanks for the PR! A few thoughts:

 - Let's call the section `Install`.
 - They probably should be installed as devDependencies? 
 - Maybe we should also add a yarn command, for a quick copy/paste.